### PR TITLE
The Witness: Add newly submitted junk hints

### PR DESCRIPTION
**Pull request by NewSoupVi**
Added junk hints submitted on the [Witness junk hint submission form](https://forms.gle/xBdpDqSmwpqvnkjs6).

Added game hints for the new games.

Reorganised game hints to be in "world folder order". It looks a little stupid but it's the easiest way for me to keep track of them. :D

### CODE REVIEW

**Review by remyjette — APPROVED** [review]
Love the new hints! Read through all the new ones, didn't see any typos

**Review by nicholassaylor — APPROVED** [review]
Very amusing hints, no typos found, LGTM

**Review by ScipioWright — APPROVED** [review]
tunc

File: worlds/witness/hints.py [hunk 1]
```diff
@@ -11,79 +11,80 @@
...
-    "Have you tried Risk of Rain 2?\nI haven't either. But I hear it's incredible!",
     "Have you tried Rogue Legacy?\nAfter solving so many puzzles it's the perfect way to rest your \"thinking\" brain.",
-    "Have you tried Secret of Evermore?\nI haven't either. But I hear it's great!",
-    "Have you tried Slay the Spire?\nExperience the thrill of combat without needing fast fingers!",
-    "Have you tried SMZ3?\nWhy play one incredible game when you can play 2 at once?",
+    "Have you tried Risk of Rain 2?\nI haven't either. But I hear it's incredible!",
+    "Have you tried Sonic Adventure 2?\nIf the silence on this island is getting to you, there aren't many games more energetic.",
     "Have you tried Starcraft 2?\nUse strategy and management to crush your enemies!",
-    "Have you tried Super Mario 64?\n3-dimensional games like this owe everything to that game.",
+    "Have you tried Shivers?\nWitness 2 should totally feature a haunted Museum.",
```
[the remark this answers is not in the document]
Comment (NewSoupVi) [reply worlds/witness/hints.py:56]: In my Defense: I'm German and we capitalise our Nouns like God intended.

File: worlds/witness/hints.py [hunk 2]
```diff
@@ -163,8 +164,23 @@
...
+    "You should try to BLJ. Maybe that'll get you through that door.",
+    "Error: Witness Randomizer disconnected from Archipelago.\n(lmao gottem)",
+    "You have found: One (1) Audio Log!\nSeries of 49! Collect them all!",
+    "In the Town area, you will find 1 good boi.\nGo pet him.",
+    "If you're ever stuck on a panel, feel free to ask Rever.\nSurely you'll understand his drawing!",
+    "[This hint has been removed as part of the Witness Protection Program]",
+    "Panel Diddle",
+    "Witness AP when",
+    "This game is my favorite walking simulator.",
+    "Did you hear that? It said --\n\nCosmic background radiation is a riot!",
```
Comment (Berserker66) [inline worlds/witness/hints.py:179]: Warframe

**Review by Berserker66 — APPROVED** [review]

**Pull request merged**